mol/molecule.py

```diff
@@ -3309,6 +3323,8 @@ def read(self, filename, inputformat=None, **other):
         if inputformat in self.__class__._readformat:
             if inputformat == "rkf":
                 return self.readrkf(filename, **other)
+            elif inputformat == "coskf":
```

Comment:
Do you have an example small coskf file you can add and create a small unit test similar to this? https://github.com/SCM-NV/PLAMS/blob/trunk/unit_tests/test_molecule.py#L1064

You will probably need to add `skip_if_no_ams_installation()` so it doesn't fail on GitHub tests if so